code/ui/components/src/tabs/tabs.stories.tsx

```diff
@@ -181,28 +188,78 @@ export const StatefulStaticWithSetBackgroundColor = {
       </TabsState>
     </div>
   ),
-};
+} satisfies Story;
 
-export const StatefulDynamic = {
-  render: (args: ComponentProps<typeof TabsState>) => (
-    <TabsState initial="test3" {...args}>
+export const StatefulDynamicWithOpenTooltip = {
+  parameters: {
+    viewport: {
+      defaultViewport: 'mobile2',
+    },
+    chromatic: { viewports: [414] },
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    await new Promise((res) =>
+      // The timeout is necessary to wait for Storybook to adjust the viewport
+      setTimeout(async () => {
+        const addonsTab = canvas.getByText('Addons');
+        fireEvent(addonsTab, new MouseEvent('mouseenter', { bubbles: true }));
+        await waitFor(() => screen.getByTestId('tooltip'));
+        res(undefined);
+      }, 500)
+    );
+  },
```

Comment:
The timeout thing is not great as it could cause some flakyness, can we try to circumvent it? Maybe this does the job? The `findByText` function might fail intially, but it will retry and succeed.

```suggestion
  play: async ({ canvasElement }) => {
    const canvas = within(canvasElement);
    const addonsTab = await canvas.findByText('Addons');
    await fireEvent(addonsTab, new MouseEvent('mouseenter', { bubbles: true }));
    await expect(await screen.findByTestId('tooltip')).toBeInTheDocument();
  },
```

Reply:
I thought you were going to change this to that other solution we saw today?

Reply:
Oops. Forget to push. Now you can see the new commit :)